src/index.js

```diff
@@ -24,7 +38,26 @@ async function startServer() {
   )
 }
 
-startServer().catch((error) => {
-  logger.info('Server failed to start :(')
-  logger.error(error)
-})
+function stopServer() {
+  if (server) {
+    server
+      .stop({ timeout: 10000 })
+      // eslint-disable-next-line promise/always-return
+      .then(() => {
+        server.logger.info('Stopped hapi server')
+        // eslint-disable-next-line n/no-process-exit
+        process.exit(0)
+      })
+      .catch((error) => {
+        server.logger.error(
+          error,
+          'Error encountered when stopping hapi server'
+        )
+        // eslint-disable-next-line n/no-process-exit
```

Comment:
worth having a read of why this is causing an error in eslint. IMO I don't think we should be ignoring eslint.
Details around why can be seen:
- https://eslint.org/docs/latest/rules/no-process-exit and then 
- https://nodejs.org/api/process.html#process_process_exitcode

Reply:
Wondering if its worth reconsidering pulse. Originally just having our own hooks on `process.on('SIGTERM')` etc seemed smaller and cleaner but it feels like we're now adding quite a bit of code with no context/docs to the template just call `server.stop()`